public/script.js

```diff
@@ -178,3 +178,15 @@ function notify(data, type){
     $(`#${notficationID}`).remove();
   });
 }
+// swipeable side bar
+$(".swipe-area").swipe({
+  swipeStatus:function(event, phase, direction, distance, duration, fingers)
+    {
+      if (phase=="move" && direction =="right") {
+         $(".left").addClass("open-menu");
+      }
+      if (phase=="move" && direction =="left") {
+         $(".left").removeClass("open-menu");
+      }
```

Comment:
In addition to these, you have to change class of `#toggle-icon` also. You have to change its class from `fa-angle-right` to `fa-angle-left` and vice-versa.

Reply:
Okay Yeah I'll do that .